### PR TITLE
[KeyboardWatcher] Add no-op test

### PR DESCRIPTION
**Pull request by romoore**

### CODE REVIEW

File: components/private/KeyboardWatcher/BUILD [hunk 1]
```diff
@@ -21,4 +24,34 @@ mdc_public_objc_library(
     deps = [
         "//components/private/Application",
     ],
+    sdk_frameworks = [
+        "CoreGraphics",
```
Comment (yarneo) [inline components/private/KeyboardWatcher/BUILD:28]: + "UIKit" ?

**Review by yarneo — APPROVED** [review]
LGTM % comment

File: components/private/KeyboardWatcher/BUILD (hunk 1 above)
Comment (romoore) [reply components/private/KeyboardWatcher/BUILD:28]: Bazel didn't complain, so I didn't add it?

Comment (yarneo) [reply components/private/KeyboardWatcher/BUILD:28]: Looks like it uses UIKit, and other BUILDs have it there so I would maybe be on the safe side and add it, but upto you, my knowledge of Bazel is limited at this point

Comment (romoore) [reply components/private/KeyboardWatcher/BUILD:28]: Added the dependency.  Generally, I follow a policy of "if the tool doesn't need it, don't use it. "

Comment (yarneo) [reply components/private/KeyboardWatcher/BUILD:28]: Taken from https://docs.bazel.build/versions/master/be/objective-c.html

screen shot 2017-12-20 at 12 02 17 pm

I guess it's a NOOP to add it or not. For consistency sake I would add it, if we want to make it cleaner, we should then remove it from all other BUILDs

**Pull request merged**